java/ql/test/experimental/query-tests/security/Clickjacking/Test.java

```diff
@@ -0,0 +1,14 @@
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletResponse;
+
+public class Test {
+
+   public void safeXframeOption(HttpServletResponse response) {
+        response.addHeader("X-Frame-Options", "DENY");  //GOOD
+    }
+
+    public void unsafeXframeOption(HttpServletResponse res) {
+        res.setHeader("X-Frame-Options", "SAMEORIGIN"); //BAD 
```

Comment:
Why is this bad? I couldn't find a recommendation that you shouldn't use it in either the [OWASP](https://cheatsheetseries.owasp.org/cheatsheets/Clickjacking_Defense_Cheat_Sheet.html) or [Mozilla](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options) documentation.